doc/01_general/04_architecture.md

```diff
@@ -0,0 +1,319 @@
+Further information can be found [here](../03_research/03_planning/Readme.md).
+
+### Preplanning
+
+Uses information from the map and the path specified by CARLA to find a first concrete path to the next intermediate
+point.
+Information from [Obstacle Detection](#Obstacle-Detection-and-Classification), [Prediction](#Prediction) etc. is not
+yet taken into account.
+
+Subscriptions:
+
+- ```map``` ([std_msgs/String](https://docs.ros.org/en/api/std_msgs/html/msg/String.html))
+- ```navigation``` (waypoints and high-level route description)
+- ```odometry``` ([nav_msgs/Odometry](https://docs.ros.org/en/api/nav_msgs/html/msg/Odometry.html))
+
+Publishes:
+
+- ```provisional_path``` ([nav_msgs/Path Message](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Path.html))
+
+### Decision Making
+
+Checks whether the path from [Preplanning](#Preplanning) actually can be taken.
+If the data from the [Perception](#Perception) indicates that the path needs to be adjusted,
+this node decides which actions to take.
+Based on this decision, the [Local path planning](#Local-path-planning) plans a new path accordingly.
+
+Subscriptions:
+
+- ```map``` ([std_msgs/String](https://docs.ros.org/en/api/std_msgs/html/msg/String.html))
+- ```navigation``` (waypoints and high-level route description)
+- ```odometry``` ([nav_msgs/Odometry](https://docs.ros.org/en/api/nav_msgs/html/msg/Odometry.html))
+- ```provisional_path``` ([nav_msgs/Path Message](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Path.html))
+- all data from [Perception](#Perception)
+
+Publishes:
+
+- ```decision``` ([std_msgs/String](https://docs.ros.org/en/api/std_msgs/html/msg/String.html))
+
+### Local path planning
+
+Translates the decisions made by the [Decision Making](#Decision-Making) into a concrete path.
+Can publish the distance to the vehicle in front to use the [Path following](#path-following)'s adaptive cruise control.
+
+Subscriptions:
+
+- ```map``` ([std_msgs/String](https://docs.ros.org/en/api/std_msgs/html/msg/String.html))
+- ```odometry``` ([nav_msgs/Odometry](https://docs.ros.org/en/api/nav_msgs/html/msg/Odometry.html))
+- ```provisional_path``` ([nav_msgs/Path Message](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Path.html))
+- all data from [Perception](#Perception)
+
+Publishes:
+
+- ```path``` ([nav_msgs/Path Message](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Path.html))
+- ```max_velocity``` ([std_msgs/Float64 Message](http://docs.ros.org/en/api/std_msgs/html/msg/Float64.html]))
+- ```distance_to_next_vehicle``` ([std_msgs/Float64 Message](http://docs.ros.org/en/api/std_msgs/html/msg/Float64.html]))
+
+## Acting
+
+The job of this component is to translate the trajectory planned by the [Planning](#Planning) component into
+steering controls for the vehicle.
+This node only takes on tasks that have lower computing time, so that a fast response of the component is ensured.
+
+Further information can be found [here](../03_research/01_acting/Readme.md).
```

Comment:
One last time :) maybe "Further information regarding the acting can be found..." would be more precise.

Reply:
Fixed in 3f8ef00137e56173f50a933439c7fa21ba3738a6